inventory_valuate_date_hls/wizards/stock_quantity_history.py

```diff
@@ -4,10 +4,4 @@
 class StockQuantityHistory(models.TransientModel):
     _inherit = "stock.quantity.history"
 
-    date_range = fields.Many2one("date.range", string="Date Range")
-    inventory_valuate_date = fields.Date(
-        string="Inventory Valuate Date",
-        related="date_range.inventory_valuate_date",
-        readonly=True,
-        store=True,
-    )
+    inventory_valuate_month = fields.Many2one("date.range", readonly=True,)
```

Comment:
I'd suggest `valuation_month_id` for the field name following the convention (i.e. the field name should end with `_id` for a many2one field.

Why readonly?